src/WhatsAppCloudApi.php

```diff
@@ -359,4 +387,16 @@ public function fromPhoneNumberId(): string
     {
         return $this->app->fromPhoneNumberId();
     }
+
+    protected function getMsgRequestExtraData(): array
```

Comment:
We need to get build the request with this extra data without create extra arguments on the Client class.

Reply:
Context ID parameters should be optional. There seems to be a an issue with Graph from 26th June, 2023. Some reply buttons and lists won't get sent, and surprisingly they won't identify the issue. Also whenever I try to report the issue and share the explorer session the issue can't be submitted with a notice to wait and try again later.

The only workaround which is hurting though is to not pass in the context id if you're replying to any message id. It's not an overall issue but having the context id mandatory isn't something I would go with.

Overall, these seem to be the long-awaited missing features of this lib